### PR TITLE
[MOBILE APP] hotfix - iOS failure on startup

### PR DESCRIPTION
**Pull request by NoahNsimbe**
#### Summary of Changes (What does this PR do?)

- [x]  Updates app dependencies

#### Status of maturity (all need to be checked before merging):

- [ ] I've tested this locally
- [ ] I consider this code done
- [ ] This change ready to hit production in its current state
- [ ] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?

- Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [Jira_card_number]() (If exists)

### CODE REVIEW

**Review by Baalmart — APPROVED** [review]
LGTM

File: mobile/android/app/build.gradle [hunk 1]
```diff
@@ -24,8 +24,8 @@ if (googleMapApiKey == null) {
     throw new GradleException("Google Maps Key not found. Define google.maps.key in the key.properties file.")
 }
 
-def googleMapApiKey_airqodev = appProperties.getProperty('google.maps.key_dev')
-if (googleMapApiKey_airqodev == null) {
+def googleMapApiKeyDev = appProperties.getProperty('google.maps.key.dev')
+if (googleMapApiKeyDev == null) {
```
Comment (NoahNsimbe) [inline mobile/android/app/build.gradle:28]: @BenjaminSsempala @Mnoble-19 take note of this

File: mobile/lib/utils/exception.dart [hunk 2]
```diff
@@ -26,12 +25,6 @@ Future<void> logException(
 
   try {
     await Future.wait([
-      FirebaseCrashlytics.instance.recordError(
-        exception,
-        stackTrace,
-        fatal: true,
-        printDetails: true,
-      ),
```
Comment (NoahNsimbe) [inline mobile/lib/utils/exception.dart:34]: temporarily disabling crashlytics due to app crashing

File: mobile/test/profile_widgets_test.dart [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        home: SignUpSection(),
+      ),
+    );
+
+    await tester.pumpAndSettle();
+    final personaliseFinder = find.text('Personalise your\nexperience');
+    final createAccountTextFinder = find.text('Create your account today and enjoy air quality updates and health tips.');
+
+    expect(personaliseFinder, findsOneWidget);
+    expect(createAccountTextFinder, findsOneWidget);
```
Comment (NoahNsimbe) [inline mobile/test/profile_widgets_test.dart:40]: signup widgets tests

File: mobile/upgrade-dependencies.sh [hunk 4]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+flutter clean
+rm -rf ios/Pods
+rm -rf ios/.symlinks
```
Comment (NoahNsimbe) [inline mobile/upgrade-dependencies.sh:4]: Script to update dependencies.

**Pull request merged**